specification/data-faas.md

```diff
@@ -0,0 +1,91 @@
+# Semantic conventions for FaaS spans
+
+This document defines how to describe an instance of a function that runs without provisioning or managing of servers (also known as serverless) with spans.
+
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [General Attributes](#general-attributes)
+  * [Difference between execution and instance](#difference-between-execution-and-instance)
+- [Function Type](#function-type)
+  * [Datasource](#datasource)
+  * [HTTP](#http)
+  * [PubSub](#pubsub)
+  * [Timer](#timer)
+  * [Other](#other)
+
+<!-- tocstop -->
+
+# General Attributes
+
+Span `name` should be set to the function name being executed. Depending on the value of the `faas.trigger` attribute, additional attributes MUST be set. For example, an `http` trigger MUST follow the [HTTP Server semantic conventions](data-http.md#http-server-semantic-conventions). For more information, refer to the [Function Type](#function-type) section.
+
+If Spans following this convention are produced, a Resource of type `faas` MUST exist following the [Resource semantic convention](data-resource-semantic-conventions.md#function-as-a-service). 
+
+| Attribute name  | Notes  and examples  | Required? |
+|---|---|--|
+| `faas.trigger` | Type of the trigger on which the function is executed. <br > It SHOULD be one of the following strings: "datasource", "http", "pubsub", "timer", or "other". | Yes |
```

Comment:
w/o knowing too much about OTel policy: 
my gut feeling is, that this generic trigger definition attempts to unify very specific entities of various faas service providers. The benefit of this unification is yet unclear. 
I would consider to introduce a faas service provider (e.g. AWS, GCF etc.) and specific triggers for these service providers.